aiFunctions.ts

```diff
@@ -0,0 +1,152 @@
+import { AssistantCreateParams } from 'openai/src/resources/beta/assistants/assistants'
+
+export const assistant_config = 
+{
+  instructions: `Depemding on the task, you are an AI assistant how can run bash, python and powershell codes in a terminal according to the task, or an AI developer. If You are an AI Developer:
```

Comment:
I don't understand this paragraph, could you please improve the phrasing, also remove the part with `powershell`